### ISSUE TITLE
Can it clone the File object?

### ISSUE THREAD

**Issue by AjaxSolutions** [lukeed/klona#42]
https://developer.mozilla.org/en-US/docs/Web/API/File

**Comment by jimmywarting** [comment]
I don't think this lib can but [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) can at least.

anyway to clone a file one would have to do: `new File([file], file.name, file)` to clone it
